modules/client-configuration/pages/supported-features-es.adoc

```diff
@@ -13,52 +13,54 @@ The icons in this table indicate:
-| Registration | icon:check[role="green"] | icon:check[role="green"]
-| Install packages | icon:check[role="green"] | icon:check[role="green"]
-| Apply patches | icon:check[role="green"] | icon:check[role="green"]
-| Remote commands | icon:check[role="green"] | icon:check[role="green"]
-| System package states | ``Salt`` | ``Salt``
-| System custom states | ``Salt`` | ``Salt``
-| Group custom states | ``Salt`` | ``Salt``
-| Organization custom states | ``Salt`` | ``Salt``
-| System set manager (SSM) | ``Salt`` | ``Salt``
-| Service pack migration | icon:times[role="danger"] | icon:times[role="danger"]
-| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"]
-| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt``
-| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional``
-| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"]
-| System deployment (PXE/Kickstart) | icon:check[role="green"] | icon:check[role="green"]
-| System redeployment (Kickstart) | ``Traditional`` | icon:check[role="green"]
+| Feature | {slsa}{nbsp}ES{nbsp}6 |  {slsa}{nbsp}ES{nbsp}7 |  {slsa}{nbsp}ES{nbsp}8
+| Client | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| Operating system packages | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| Registration | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| Install packages | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| Apply patches | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| Remote commands | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| System package states | ``Salt`` | ``Salt`` | ``Salt``
+| System custom states | ``Salt`` | ``Salt`` | ``Salt``
+| Group custom states | ``Salt`` | ``Salt`` | ``Salt``
+| Organization custom states | ``Salt`` | ``Salt`` | ``Salt``
+| System set manager (SSM) | ``Salt`` | ``Salt`` | ``Salt``
+| Service pack migration | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
+| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"] | icon:check[role="green"]
+| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt`` | ``Salt``
+| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional`` | ``Traditional``
+| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"] | icon:check[role="green"]
+| System deployment (PXE/Kickstart) | icon:check[role="green"] | icon:check[role="green"] | icon:check[role="green"]
+| System redeployment (Kickstart) | ``Traditional`` | icon:check[role="green"] | icon:check[role="green"]
```

Comment:
v8: Not tested and we know it is not working. Somebody need to invest some time to fix it before we publish this.